Bottom: 77e42da04dc7e521f2e5ab569ae8779d6cf6e296
Top:    75bd931abba7fce0516a532ed0c59a7697be6d60
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-07-01 16:22:32 +0800

Refresh of tun-rework-on-flow-cache-0

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 91f0d79..ba26891 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -162,7 +162,7 @@ struct tun_flow_table {
 #define TUN_FLOW_TABLE_SIZE(_num) (offsetof(struct tun_flow_table, ents[_num]))
 
 #define TUN_NUM_FLOW_ENTRIES 1024
-#define TUN_FLOW_MASK 0x3FF
+#define TUN_FLOW_MASK (TUN_NUM_FLOW_ENTRIES - 1)
 
 /* Since the socket were moved to tun_file, to preserve the behavior of persist
  * device, socket filter, sndbuf and vnet header size were restore when the
